pkg/member/participant.go

```diff
@@ -87,10 +85,9 @@ func (m *Participant) InitInfo(p participant, rootPath string, leaderName string
 	m.member = p
 	m.memberValue = string(data)
 	m.rootPath = rootPath
```

Comment:
Do we still need it?

Reply:
Yes, this issue does not close. `getMemberLeaderPriorityPath` still uses `rootPath`.

I will replace them one by one and try to remove `rootPath`